pkg/controller/risingwave_controller.go

```diff
@@ -108,7 +142,7 @@ func (c *RisingWaveController) Reconcile(ctx context.Context, request reconcile.
 	// Build a workflow and run.
 	workflow := c.reactiveWorkflow(risingwaveManager, &mgr)
 
-	updateRisingWaveStatus := mgr.NewAction("UpdateRisingWaveStatusViaClient", func(ctx context.Context, l logr.Logger) (ctrl.Result, error) {
+	updateRisingWaveStatus := mgr.NewAction(RisingWaveAction_UpdateRisingWaveStatusViaClient, func(ctx context.Context, l logr.Logger) (ctrl.Result, error) {
```

Comment:
Why add the `RisingWaveAction_` as the prefix?

Reply:
To make sure they are clear enough to be identified as the action names of RisingWave. Think about the situation: we have multiple controllers in the same package, and there will be constants for different CRs.